src/main/java/seedu/address/logic/commands/project/FilterCommand.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.logic.commands.project;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TASK_FILTER_BY_ASSIGNEE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TASK_FILTER_BY_DEADLINE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TASK_FILTER_BY_NAME;
+
+import java.util.function.Predicate;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.project.Project;
+import seedu.address.model.task.Task;
+
+/**
+ * Filters tasks by assignee's name, task name or deadline.
+ */
+public class FilterCommand extends Command {
+    public static final String COMMAND_WORD = "filter";
+
+    public static final String MESSAGE_NOT_IMPLEMENTED_YET = "Filter command not implemented yet";
```

Comment:
You didn't implement it yet?

Reply:
I forgot to delete it, I will delete it later